### DISCUSSION TITLE
SvelteKit example

### DISCUSSION THREAD

**Discussion by DePasqualeOrg** [teamhanko/hanko#289] [Ideas]
SvelteKit is now in release candidate phase, with no more breaking changes planned until 1.0. Perhaps this would be a good time to create an example for the docs using SvelteKit.

**Comment by FlxMgdnz** [comment]
We agree. Unfortunately there is an [issue](https://github.com/teamhanko/hanko/issues/270) with [preact-custom-element](https://github.com/preactjs/preact-custom-element) and Svelte.

At the moment we are waiting for a reaction from the project's maintainers. If nothing happens in the next few weeks, we will start thinking about other options.

**Comment by FlxMgdnz** [comment]
We decided to take matters into our own hands and did a fork of the preact-custom-element lib to fix the issue. See #424
